src/main/java/org/vietabroader/view/MainView.java

```diff
@@ -293,35 +307,41 @@ private JPanel createColumnPanel() {
         c.anchor = GridBagConstraints.CENTER;
         btnRefresh.setPreferredSize(new Dimension(100, 50));
         panelMain.add(btnRefresh, c);
+        sheetFreshController.setButtonRefresh(btnRefresh);
 
         return panelMain;
     }
 
-    private JPanel createOneColumn(String label) {
-        final JTextField txtCol = new JTextField("A",5);
-        JPanel panel = new JPanel() {
-            @Override
-            public void setEnabled(boolean enabled) {
-                super.setEnabled(enabled);
-                txtCol.setEnabled(enabled);
-            }
-        };
-        panel.setLayout(new GridBagLayout());
-        GridBagConstraints c = new GridBagConstraints();
-
-        c.weightx = 1;
-
-        c.gridy = 0;
-        c.anchor = GridBagConstraints.CENTER;
-        panel.add(new JLabel(label), c);
-
-        c.gridy = 1;
-        c.fill = GridBagConstraints.HORIZONTAL;
-        txtCol.setInputVerifier(new ColumnVerifier());
-        txtCol.setToolTipText("Enter a valid column name. Ex: A, AB, ...");
-        panel.add(txtCol, c);
+    private class oneColumn {
```

Comment:
A better design would be letting this class extend JPanel, and create a `getTextField` method. 
Also, name of a class should be capitalized at the first letter.